### DISCUSSION TITLE
Unable to bill user after called "requestSubscription"

### DISCUSSION THREAD

**Discussion by paul-tam** [hyochan/react-native-iap#2206] [Q&A]
After called "requestSubscription", the google play billing library pop-up has shown with the subscription plan's information. When user has chosen the payment method, let say "Visa" and proceed to finish the subscription, Google play billing library has returned the following error.

`Your transaction can't be completed. To continue this transaction, use a different payment instrument or contact us. Learn more.`
I have asked google for help several times, but the transaction is always fail. Is anybody facing the same issue with me?

react-native-iap: 9.0.4
react-native: 0.66.5
Platform: Android 11 (Samsung Galaxy S10)

**Comment by andresesfm** [comment]
I don't think this is related to this library, you might be better off posting on Stack Overflow or Google support
